liveblog/liveblog
# [LBSD-2849] - Patch error on poll voting

### Purpose

This PR introduces a retry mechanism on poll voting to avoid potential ETag mismatches. This can occur if another client updates the poll between the time the client fetches the poll and the time it tries to send the patch request.

### What has changed

- Added retry mechanism in `placeVote` function.
- Added check in `placeVote` function to also check if voting on the selected poll has also happened on another tab
- Added a `beta` icon on the Poll feature on the Editor side

### Steps to test

1. Log in to the Editor admin.
2. Create a new poll.
3. Open the Live preview of the blog in 3 different browsers.
4. Try voting in the 3 browsers and observe that voting happens as expected.

Resolves: [LBSD-2849](https://sofab.atlassian.net/browse/LBSD-2849)
